tools/lkl/tests/net.sh

```diff
@@ -43,15 +43,19 @@ fi
 if ! [ -z $DST ]; then
     echo "== RAW socket (LKL net) tests =="
     sudo ip link set dev ${IFNAME} promisc on
-    sudo ./net-test raw ${IFNAME} ${DST} dhcp
+    sudo ./net-test raw ${IFNAME} ${DST} dhcp &
+    sleep 5
+    sudo killall -q net-test && (echo "killed, skipped")
```

Comment:
ah, that's a good idea.
maybe we can introduce LKL_TEST_DHCP for circleci environment, then timeout should not be needed.

will resubmit a patch.